pkg/broker/broker.go

```diff
@@ -769,8 +770,38 @@ func (a AnsibleBroker) isJobInProgress(instance *apb.ServiceInstance,
 	return len(methodJobs) > 0, token, nil
 }
 
+func (a AnsibleBroker) GetBind(instance apb.ServiceInstance, bindingUUID uuid.UUID) (*BindResponse, error) {
+
+	log.Debug("XXX entered GetBind")
+
+	// this might not be required
```

Comment:
I like your honesty. Why might this not be required?  Is this to verify that a provision has completed?